index.js

```diff
@@ -15,12 +15,13 @@ const instrumentHTTP = require('./instrumentation/http');
 /**
  * Configure honeycomb with our standard preferences
  * @param {string} name the name of the service
+ * @param {string | "http"} process the type of process (e.g. "http" or "worker")
  * @param {string} gitSha the git sha representing the version of the service
  * @param {Options | "mock"} options configuration options, or the literal "mock" to use the mock backend
  *
  * @returns {beeline.Beeline}
  */
-module.exports = function setup(name, gitSha, options) {
+module.exports = function setup(name, process = 'http', gitSha, options) {
```

Comment:
Any reason this is the second arg? If it was optional and last wouldn't it mean we could update this package and nothing would break if the updater didn't notice the change (although maybe we only use this package in heirloom? so it doesn't really matter?)

Reply:
That's a good point! Is there a convention around adding new args to a method and where they should go?
I added it after `name` since it's related to the app, but there's no functional reason not to put it at the end of the list for the reason you suggested

Reply:
I think it's typical to put optional parameters at the end of the argument list https://stackoverflow.com/questions/2896106/why-optional-parameters-must-appear-at-the-end-of-the-declaration